### DISCUSSION TITLE
Re-verification process can never be done

### DISCUSSION THREAD

**Discussion by caida-ricky** [community/community#48725] [GitHub Education]
I upload my paystub with all the information required and updated my short bio to include my full time and affiliation. However, I could not re-verify my github education status.

**Comment by Siunaus** [comment]
same issue

**Reply by zhao605207225** [reply]
ye,same issue

**Comment by samus-aran** [comment]
Hi @caida-ricky  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by caida-ricky** [comment]
well. the error message is not informative. I followed the suggestions but no luck at all.

![image](https://user-images.githubusercontent.com/27710684/222604624-5bee30ee-92f6-4cfa-807b-23658df29a6e.png)

**Reply by Butanediol** [reply]
exact same error here.

opened two support tickets, still not reply after 1 week
